solr/contrib/ltr/src/java/org/apache/solr/ltr/FeatureLogger.java

```diff
@@ -192,6 +192,33 @@ public String makeFeatureVector(LTRScoringQuery.FeatureInfo[] featuresInfo) {
       return features;
     }
 
+    /*
```

Comment:
If all the tests using these methods live in the same package maybe we could put the methods there without `public` so that visibility is restricted to package level?